playground/webpack.config.js

```diff
@@ -10,7 +10,8 @@ const postcssShopify = require('postcss-shopify');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
   .BundleAnalyzerPlugin;
 
-const ICON_PATH_REGEX = /icons\//;
+const INTERNAL_ICON_PATH_REGEX = /icons\//;
```

Comment:
Anything that matches `icons\/` would also match anything that matches `polaris-icons\/` so I don't think this second regex is needed.

If we wanted to be stricter and match an exact folder name we could still do it in single regexp I think.  Something like `/\/(icons|polaris-icons)\//` would match both `blah/icons/blah` and `blah/polais-icons/blah`.